### ISSUE TITLE
Tooltips from navigation bar are displayed without hovering over after keyboard navigation

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5290]
**Important details:**

* Tooltips from the navigation bar are also displayed on the “Get started” screen after signing out;

**Affected versions:**

* 2.13.0 (2.202212210602); 

**Tested Platforms:**

* MacOS 12.6.1;

* Windows 11;

**Prerequisites:**

* Have Mozilla VPN installed and open;

* Have a Fx account with an active subscription;

* Be signed in;

**Steps to reproduce:**

1. Press the “Tab” key until the icons from the navigation bar are selected;
1. Minimize the app / click outside the VPN screen;
1. Restore the app / click anywhere on the VPN screen (except on the navigation bar);
1. Observe the navigation bar;

**Expected result:**

* There is no tooltip displayed on the the navigation bar as no hovering was performed;

**Actual result:**

* The previously selected icon with the keyboard from the navigation bar displays the specific tooltip without hovering over;

**Notes:**

* See the attached video;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-3548)
┆Reporter: Bianca Hidecuti

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Bianca Hidecuti Are you sure this is the right video? From the description it looks like the issue is that once an element is selected (via tabs, or just being the last one clicked), minimizing and maximizing will generate the text label to show up again. I can reproduce that.

But the video shows that when you sign out you also get the text label showing up. I can’t reproduce that on 2.12, but I’m not sure if this video was incorrectly uploaded, or you indeed are reporting this is happening in 2.13. Can you please clarify?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Santiago Andrigo The video is correct. It shows 2 scenarios:

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Thanks Valentina. Bumping this to Medium as this happens regularly and in a variety of screens – e.g. https://mozilla-hub.atlassian.net/browse/VPN-3562 ( https://mozilla-hub.atlassian.net/browse/VPN-3562|smart-link ). Lesley Norton for thoughts on the underlying cause.

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Lesley Norton, could you please add the fixversion? Thank you.

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Verified this as fixed on the latest 2.13 builds (2.13.0 (2.202301161600)) from the release branch, using MacOS 12.6.1 and Windows 11 → Tooltips from the navigation bar are no longer displayed on the “Get started” screen after signing out, after previously using the keyboard to navigate. 

Attaching postfix video.

I also added the 2.13 fix version, as the fix is merged into the release branch.

!vpn-3548.mov|width=1274,height=1924!
